addons/hr_timesheet/views/hr_timesheet_views.xml

```diff
@@ -22,14 +22,14 @@
             <field name="name">account.analytic.line.tree.hr_timesheet</field>
             <field name="model">account.analytic.line</field>
             <field name="arch" type="xml">
-                <tree editable="top" string="Timesheet Activities" sample="1">
+                <tree editable="bottom" string="Timesheet Activities" sample="1">
                     <field name="date"/>
                     <field name="employee_id" invisible="1"/>
                     <field name="project_id" required="1" options="{'no_create_edit': True}"/>
-                    <field name="task_id" optional="show" options="{'no_create_edit': True, 'no_open': True}" widget="task_with_hours" context="{'default_project_id': project_id}" domain="[('project_id', '=', project_id)]"/>
+                    <field name="task_id" optional="show" options="{'no_create_edit': True, 'no_open': True}" widget="task_with_hours" context="{'default_project_id': project_id}"/>
```

Comment:
If the goal is to allow only not private tasks to be selected, shouldn't we keep a domain like `['project_id', '!=', False]`?

Reply:
May I also ask you to rephrase the commit part:
```
- remove extra domain from task_id field in tree view show user
  user able to select all non private tasks.
```
into:
```
- ensure that all non private tasks can be selected in timesheet tree view.
```

Reply:
the domain is already given in the task_id field definition from python [line](https://github.com/odoo/odoo/blob/master/addons/hr_timesheet/models/hr_timesheet.py#L42). Thanks :slightly_smiling_face:

Reply:
> the domain is already given in the task_id field definition from python [line](https://github.com/odoo/odoo/blob/master/addons/hr_timesheet/models/hr_timesheet.py#L42). Thanks 🙂

Ok great 🙂 Thanks!

Reply:
Can you still rephrase the commit as proposed? Thanks!

Reply:
@IT-Ideas We should also remove it from the form view. [Community Form View](https://github.com/odoo/odoo/blame/master/addons/hr_timesheet/views/hr_timesheet_views.xml#L131)

Reply:
> @IT-Ideas We should also remove it from the form view. [Community Form View](https://github.com/odoo/odoo/blame/master/addons/hr_timesheet/views/hr_timesheet_views.xml#L131)

That would make sense, maybe check with @aju-odoo .
Thanks 🙂